playwright-staging/tests/submit/erp.spec.js

```diff
@@ -0,0 +1,100 @@
+// @ts-check
+const { expect } = require('@playwright/test');
+const { test } = require('../../browserstack');
+const { Commands } = require('../utils/commands');
+const { ErpNextTester } = require('@comicrelief/test-utils/ErpNextTester');
+const Chance = require('chance');
+const chance = new Chance();
+
+const erpTester = new ErpNextTester(
+  process.env.ERP_HOST,
+  process.env.ERP_API_KEY,
+  process.env.ERP_API_SECRET,
+);
+
+test('Verify giftaid submit ERP data @regression', async ({ page }) => {
+  const commands = new Commands(page);
+  const rndmString = await commands.randomString();
+  
+  const contactInfo = {
+    mobile: chance.phone({ country: 'uk', mobile: true }).replace(/\s/g, ''), // Remove spaces from the phone number
+    firstName: `test${rndmString}`,
+    lastName: chance.last(),
+    email: `giftaid-staging-${chance.email()}`,
+    postcode: chance.postcode(),
+    address1: chance.address(),
+    address2: chance.street(),
+    address3: 'test address 3',
+    town: chance.city()
+  };
+  
+  // Navigate to the giftaid page
+  await page.goto(process.env.BASE_URL, { timeout: 30000 });
+  await page.waitForLoadState('domcontentloaded');
+  
+  // Click the Giftaid checkbox
+  await page.click('#field-label--giftaid');
+  
+  // Populate all the form fields with valid inputs
+  await commands.populateFormFields(page, contactInfo);
+  
+  // Select marketing preferences as required
+  await commands.selectMarketingPrefs(page);
+  
+  // Submit the form and wait for the navigation to ensure the submission goes through
+  await Promise.all([
+    page.waitForNavigation(), // This ensures that the navigation happens before the check
+    page.click('button[type=submit]')
+  ]);
+  
+  // Check for the thank you message to confirm successful submission
+  await expect(page.locator('div > h1')).toHaveText(`Thank you, ${contactInfo.firstName}!`);
```

Comment:
Just a general thought (not specifically for this test or repo), but it might be good to use more specific selectors, something like `div.success-wrapper--inner h1` or even `div.success-wrapper--inner h1:first-child`, just to make things as non-brittle as possible